LICENSE.md

```diff
@@ -1,21 +1,10 @@
 MIT License
 
-Copyright (c) 2012  Chun-hao Hu Copyright (c) 2016-present Multiple Contributors
+Copyright (c) 2012 Chun-hao Hu
+Copyright (c) 2016-present Multiple Contributors
 
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
```

Comment:
Are you sure that avoid the 80 chars wordwrap is the suitable here?

Reply:
You're right, I will restore it.